CONTRIBUTING.md

```diff
@@ -7,6 +7,10 @@ We need help with this project, so contributions are welcome.
 Conduct collaboration around u-root in accordance to the [Code of
 Conduct](https://github.com/u-root/u-root/wiki/Code-of-Conduct).
 
+## Slack Channel
+
+You can sign up for the Slack channel [here](http://slack.u-root.com:3000/).
```

Comment:
I like the idea in general! I tried this url out this morning and it didn't seem to work for me. Would probably be good to use https here.